.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
It seems that you accidentally committed a file from your editor configuration (`.vscode/settings.json`). Please delete it from the PR.

Reply:
oops, sorry for the inattention... Fixed!